<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Go_B">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Home_RB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>125</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Paths_RBs</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="Work_RB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>125</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>WORK</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Paths_RBs</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Paths_RBs"/>
 </buttongroups>
</ui>
